Debugging Fundamentals/Task1/Product.cs

```diff
@@ -7,6 +9,19 @@ public Product(string name, double price)
             Name = name;
             Price = price;
         }
+        public override bool Equals(object productObject)
```

Comment:
incomingProduct looks good.
Don't use words like 'Object', 'class' in names. Names should look meaningful and real.

Reply:
fixed
![image](https://github.com/user-attachments/assets/0b8adf3c-28e8-4464-a653-cb74ff30fa30)